diff --git a/EVE_target.h b/EVE_target.h
index af7d5ec..109d0ac 100644
--- a/EVE_target.h
+++ b/EVE_target.h
@@ -748,10 +748,12 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR TH
                #endif


-               #define EVE_CS_PORT GPIOD
-               #define EVE_CS GPIO_PIN_12
+               #define EVE_CS_PORT GPIOA
+               #define EVE_CS GPIO_PIN_4
+#if FTR_HAS_PDN
                #define EVE_PDN_PORT GPIOD
                #define EVE_PDN GPIO_PIN_13
+#endif
                #define EVE_SPI SPI1
                #define EVE_DMA_INSTANCE DMA2
                #define EVE_DMA_CHANNEL 3
@@ -771,12 +773,16 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR TH

                static inline void EVE_pdn_clear(void)
                {
-                       HAL_GPIO_WritePin(EVE_PDN_PORT, EVE_PDN, GPIO_PIN_SET);
+#if FTR_HAS_PDN
+                       // HAL_GPIO_WritePin(EVE_PDN_PORT, EVE_PDN, GPIO_PIN_SET);
+#endif
                }

                static inline void EVE_pdn_set(void)
                {
-                       HAL_GPIO_WritePin(EVE_PDN_PORT, EVE_PDN, GPIO_PIN_RESET);
+#if FTR_HAS_PDN
+                       // HAL_GPIO_WritePin(EVE_PDN_PORT, EVE_PDN, GPIO_PIN_RESET);
+#endif
                }
